### PR TITLE
initial list of projects

### PR DESCRIPTION
**Pull request by aaarendt**
This PR lists our existing projects, with links to those that have repos already created. I'd welcome feedback on the table format and if the descriptions need refinement.

### CODE REVIEW

File: book/projects/list_of_projects.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
+| [crunchy-snow](https://github.com/geo-smart/crunchy-snow) | Seasonal snow provides drinking water for billions, but current global measurements of snow depth lack adequate spatial and temporal resolution for effective resource management–especially in mountainous terrain. Recent work has demonstrated the potential to retrieve snow-depth measurements from Sentinel-1 synthetic aperture radar (SAR) backscatter data. However, comparisons with airborne lidar data suggest that existing algorithms fail to capture the full complexity of relationships between snow depth, terrain, vegetation, and SAR backscatter, the physics of which are poorly understood. We suggest that a neural network may be able to effectively learn these relationships and retrieve snow depth from SAR backscatter with improved accuracy. | Quinn Brencher, Eric Gagliano |
+| [Spacetime-elevation](https://github.com/geo-smart/spacetime-elevation) | Using Gaussian Process regression to compute continuous surface change from spatially and temporally sparse elevation measurements (applications: glacier elevation changes, snow depth).  | Romain Hugonnet |
```
Comment (friedrichknuth) [inline book/projects/list_of_projects.md]: Replace "Spacetime" with "spacetime"

File: book/projects/list_of_projects.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
 
-Here is where we will be listing the projects during the {{dates}} {{ hackweek }} hackweek:
+Here is our current list of project for our {{dates}} {{ hackweek }} hackweek:
 
-| Project Name (with link to GitHub repo) | slack channel | final presentation | 
+| Project Name (with link to GitHub repo) | Short Description | Project Lead(s) | 
 |:--------|:--------|:-----|
-|  |  |   |
\ No newline at end of file
+| [snowcover](https://github.com/geo-smart/snowcover) | The Normalized snow difference index has been traditionally  being used to identify snow in satellite imagery. In this project we will explore ML models to derive snow covered areas from satellite imagery and will validate it against ASO imagery. | Aji John |
```
Comment (friedrichknuth) [inline book/projects/list_of_projects.md]: Remove space after "traditionally"

Comment (friedrichknuth) [inline book/projects/list_of_projects.md]: Replace "it" with "them"

File: book/projects/list_of_projects.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+| [Snow-Extrapolation](https://github.com/geo-smart/Snow-Extrapolation) | Goal: Improve NSM prediction performance in the Sierra Nevada mountains through domain constraints and the exploration of different ML algorithms. | Ryan Johnson |
```
Comment (friedrichknuth) [inline book/projects/list_of_projects.md]: Define "NSM"

**Review by friedrichknuth — APPROVED** [review]
Some suggested minor edits to the descriptions. Table format looks great!

**Review by spestana — APPROVED** [review]
Looks good to me! And thanks @friedrichknuth for the updates too

**Pull request merged**